src/web/app/pages-static/request-page/request-page.component.scss

```diff
@@ -0,0 +1,5 @@
+.empty-field-placeholder::after {
+    content: "(empty)";
+    opacity: 0.5;
+    font-style: italic;
+}
```

Comment:
This is a nit: let's add the file's trailing newline.

Reply:
Strange, the trailing newline shows up on my end but doesn't appear on GH. I'll try deleting and re-adding the file

Reply:
Ok, deleting and re-adding the file seems to have fixed it